pyproject.toml

```diff
@@ -2,6 +2,13 @@
 requires = ["setuptools >= 65.5.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[tool.setuptools]
+include-package-data = false
+
+[tool.setuptools.packages.find]
+include = ["yoyodyne*"]
+exclude = ["examples*"]
+
```

Comment:
I think you need to add `wandb >= 0.15.3` as a requirement to pyproject.toml too.